Header header

#float32[] velocity
#float32[] current
#float32[] temperature
int32[] internal_load_cell_force_digital
int32[] external_load_cell_force_digital
int32[] error
